client/persistence/dexie/dexie.indexing.worker.ts

```
@@ -0,0 +1,582 @@
+          resolve,
+          reject
+        );
+      };
+
+      upgradeRequest.onerror = () => {
+        reject(upgradeRequest.error);
+      };
+    } catch (error) {
+      reject(error);
+    }
+  });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)